README.md

```diff
@@ -1,9 +1,19 @@
 # Northcoders News API
 
-For instructions, please head over to [L2C NC News](https://l2c.northcoders.com/courses/be/nc-news).
+--- 
 
+This portfolio project was created as part of a Digital Skills Bootcamp in Software Engineering provided by [Northcoders](https://northcoders.com/)
 
+---
 
---- 
+Follow these instructions if you wish to clone the repo:
 
-This portfolio project was created as part of a Digital Skills Bootcamp in Software Engineering provided by [Northcoders](https://northcoders.com/)
+## 01 Project Setup
+
+1.  Install dotenv dependency with command: `npm install dotenv`
```

Comment:
Instead of installing just dotenv, you'd instead want a user to just use `npm install` because that will install every dependency mentioned in the package.json. If they forgot this step then the server wouldn't work as we need express and all the other npm packages used!